<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-MQDCCMD" name="tests.Runner5Parallel" tests="7" failures="3" timestamp="2022-09-08T08:49:07 IST" time="34.969" errors="0">
  <testcase name="runScenario" time="4.995" classname="tests.Runner5Parallel"/>
  <system-out/>
  <testcase name="runScenario" time="5.003" classname="tests.Runner5Parallel"/>
  <system-out/>
  <testcase name="runScenario" time="4.982" classname="tests.Runner5Parallel"/>
  <system-out/>
  <testcase name="runScenario" time="5.009" classname="tests.Runner5Parallel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs2.method3(StepDefs2.java:88)
at ✽.status code is "200" as per "duplicate" and content type is "application/json"(file:///D:/api_restassured/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="4.984" classname="tests.Runner5Parallel"/>
  <system-out/>
  <testcase name="runScenario" time="4.986" classname="tests.Runner5Parallel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs2.method3(StepDefs2.java:88)
at ✽.status code is "400" as per "wrong" and content type is "application/json"(file:///D:/api_restassured/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="5.010" classname="tests.Runner5Parallel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs2.method1(StepDefs2.java:43)
at ✽.status code is "400" as per "blank" in json response body(file:///D:/api_restassured/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- tests.Runner5Parallel -->
